src/config/field-overrides/predictor.js

```diff
@@ -152,6 +152,7 @@ export default new Map([
                 message: 'expression_is_required',
             },
         ],
+        unique: false
```

Comment:
These overrides needs to be documented in a comment with a link to the JIRA issue that explains why we are overriding this.